### PR TITLE
TST: added test for _to_voxel_tolerance.

### PR DESCRIPTION
**Pull request by jchoude**
Added the test that was missing in PR #678

### CODE REVIEW

**Comment by arokem** [comment]
Thanks! Going to find a hat to eat.

File: dipy/tracking/tests/test_utils.py [hunk 1]
```diff
@@ -73,6 +74,28 @@ def test_density_map():
...
+    transfo = np.array([[1.0, 0.0, 0.0],
+                        [0.0, 1.0, 0.0],
+                        [0.0, 0.0, 1.0]])
+
+    # Offset is computed by _mapping_to_voxel. With a 1x1x1 dataset
+    # having no translation, the offset is half the voxel size, i.e. 0.5.
+    offset = np.array([0.5, 0.5, 0.5])
+
+    # Without the added tolerance in _to_voxel_coordinates, this streamline
+    # should raise an Error in the call to _to_voxel_coordinates.
```
Comment (arokem) [inline dipy/tracking/tests/test_utils.py]: Can we test that with an `assert_raises` ?

Comment (arokem) [inline dipy/tracking/tests/test_utils.py]: Oh - sorry - I see that makes no sense.

All good. I'll wait for travis to run this and then merge. 

Again - thanks for filling that in!

**Comment by Garyfallidis** [comment]
Much luv! Thx @jchoude!

**Pull request merged**